integration/feature/editing/test/src/MultiLevelBuildTests.scala

```diff
@@ -62,8 +62,10 @@ object MultiLevelBuildTests extends IntegrationTestSuite {
         val frameWatched = frame
           .evalWatched
           .map(_.path)
-          .sorted.filter(_.startsWith(wsRoot))
+          .filter(_.startsWith(wsRoot))
           .filter(!_.segments.contains("mill-launcher"))
+          .distinct
+          .sorted
```

Comment:
Added a `distinct`. It's a consequence of having the redundant `T.input` on `MillBuildRootModule.sources`.

Sort is expensive, so we do it after filtering.

Reply:
Removed the distinct again